daphne/src/taskprov.rs

```diff
@@ -223,15 +223,19 @@ impl DapQueryConfig {
 }
 
 impl VdafConfig {
-    fn try_from_taskprov(task_id: &TaskId, var: VdafTypeVar) -> Result<Self, DapAbort> {
-        match var {
-            VdafTypeVar::Prio2 { dimension } => Ok(VdafConfig::Prio2 {
+    fn try_from_taskprov(
+        task_id: &TaskId,
+        version: DapVersion,
+        var: VdafTypeVar,
+    ) -> Result<Self, DapAbort> {
+        match (version, var) {
+            (.., VdafTypeVar::Prio2 { dimension }) => Ok(VdafConfig::Prio2 {
                 dimension: dimension.try_into().map_err(|_| DapAbort::InvalidTask {
                     detail: "dimension is larger than the system's word size".to_string(),
                     task_id: *task_id,
                 })?,
             }),
-            VdafTypeVar::NotImplemented { typ, .. } => Err(DapAbort::InvalidTask {
+            (.., VdafTypeVar::NotImplemented { typ, .. }) => Err(DapAbort::InvalidTask {
                 detail: format!("unimplemented VDAF type ({typ})"),
                 task_id: *task_id,
             }),
```

Comment:
You're matching on the version but you never use it's value :thinking:

Reply:
This will get used when we add new VDAFs for taskprov. We want to be able to opt-out for any VDAF other than Prio2 in draft02.

If you like I can remove this change from this PR. (I'll add it back in a future PR, probably later this week.)

Reply:
if it's gonna be used soon then let it be, don't want to give you any more work